superset/assets/src/components/FilterableTable/FilterableTable.jsx

```diff
@@ -266,6 +325,15 @@ export default class FilterableTable extends PureComponent {
     );
   }
 
+  renderTableCell({ cellData }) {
+    const cellNode = String(cellData);
+    const jsonObject = getJsonStringifiedObject(cellData);
```

Comment:
Given that this will run for each cell (easily hundreds of thousands of times), I'm worried that trying `JSON.parse` might be expensive.

Doing a cheap proxy first like:
```javascript
cheapCouldBeJson(value) {
    return (typeof(value) === 'string' && ['{', '[', '"'].indexOf(value.substring(0, 1)))
}
```

Reply:
Done.